flink-connector-kafka/pom.xml

```diff
@@ -239,7 +239,6 @@ under the License.
 		<dependency>
 			<groupId>org.yaml</groupId>
 			<artifactId>snakeyaml</artifactId>
```

Comment:
Nit: Can we delete this dependency block completely? To me, it looks like snakeyaml is already on the classpath (by some transitive dependency) and we are adding it here unnecessarily.